### ISSUE TITLE
[ED] [Overview page] -- Complex table for screen reader users

### ISSUE THREAD

**Issue by rperez030** [w3c/wai-curricula#337]
In the contents section, the 4 modules and their content are presented in a complex table. The concern is that such structure can make the content unnecessarily difficult to consumed by someone using a screen reader since it cannot just be read line by line. I wonder if a more simpler structure such as a simple table or lists can be used. CSS can then be used to maintain the current visual layout. Since the content can be read, I do not see that as a reason for not publishing the page. This is just an observation for the editors.

**Comment by daniel-montalvo** [comment]
Thanks @rperez030 

We tried other structures, such as simple tables or nested lists, but we dismissed them because they were not useful for the main audience purpose. We simplified the table in this new iteration, and we may eventually substitute it with interactive controls for visitors to pick the modules they want based on role or other consideration.

At the moment, we think the table allows to quickly review which modules are available and which are not.

I will label this as an enhancement we need to look into in future iterations.

**Comment by daniel-montalvo** [comment]
Hey @rperez030 

We finally took a pass at this. You can see a preview at
https://deploy-preview-339--wai-curricula.netlify.app/curricula/

- The Foundation Modules are no longer a table from a screen readers perspective.
- The remaining table is a table with two rows and three columns, with "Developers", "Designers", and "Authors" as column headers.

Will be closing this once merged. Feel free to reopen if this has not been sufficiently addressed.

**Comment by rperez030** [comment]
Excellent. Thank you.
